Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1.4.0
+
+FROM node:22-alpine
+
+WORKDIR /app
+
+EXPOSE 3000
+
+COPY --link . .
+
+RUN npm install
+RUN npm run build
+
+CMD ["npm","run","serve"]
```

Comment:
Only the leading and trailing spaces were to be removed, there should still be spaces after the commas. It should read
```
CMD ["npm", "run", "serve"]
```